src/utils.js

```diff
@@ -234,3 +257,189 @@ export function trimContentDam(path) {
 
     return trimRight(trimmed, ['/']);
 }
+
+/**
+ * Normalizes a path by ensuring it only contains forward slashes abd does not end with a
+ * slash. If the given path is falsy then the method will return an empty string.
+ * @param {string} path An item's full path.
+ * @returns {string} Normalized version of a path.
+ */
+export function normalizePath(path) {
+    let normPath = path;
+    if (normPath) {
+        normPath = normPath.replace(/\\/g, '/');
```

Comment:
I think the goal here is to normalize path separators? 
Maybe rather use the `path.delimiter` function from node rather than hardcoding it?
See https://nodejs.org/api/path.html#path_path_delimiter

Reply:
Also, `path.normalize` may also be helpful in that case.
See https://nodejs.org/api/path.html#path_path_normalize_path

Reply:
Good catch, but as noted in the method description, this method purposely only uses forward slashes because it's often used to build URL paths.